### ISSUE TITLE
Plugin isn't exporting, giving an [MSUnarchiver decodeObjectForKey:]: cannot decode object of class (MSImmutableSharedStyleContainer) for key (layerStyles) error

### ISSUE THREAD

**Issue by mochabcha** [avocode/avocode#630]
I made a few image changes in the document.  The last time exported it was a couple days ago with no issue.  After changing out a few images, I get an error.

I also setup a connection between my account and Slack.

IMPORTANT: Please try to describe how did it happen.

-----
Sketch Exporter 2.1.5 running in Sketch 3.5.1
```
Error: Error: Command failed: 2016-01-27 09:46:50.117 sketchtool[4751:935630] Error: caught exception whilst suppressing update events: *** -[MSUnarchiver decodeObjectForKey:]: cannot decode object of class (MSImmutableSharedStyleContainer) for key (layerStyles)

**Comment by mochabcha** [comment]
Error: Command failed: 2016-01-27 09:55:02.852 sketchtool[4783:1044385] Error: caught exception whilst suppressing update events: *** -[MSUnarchiver decodeObjectForKey:]: cannot decode object of class (MSImmutableSharedStyleContainer) for key (layerStyles); the class may be defined in source code or a library that is not linked
2016-01-27 09:55:02.853 sketchtool[4783:1044385] Error: caught exception whilst decoding (version 76):*** -[MSUnarchiver decodeObjectForKey:]: cannot decode object of class (MSImmutableSharedStyleContainer) for key (layerStyles); the class may be defined in source code or a library that is not linked
Failed to open document file:///Volumes/Chanspace/Google%20Drive/Ingenuity%20Projects/Hass/WebProt_Family_Hass.sketch
((null) MSDocumentDomain:11)

This is the full error, it happened during Step 3 while the document was uploading to Avocode.

**Comment by joelvfernandez** [comment]
Having the same error during Step 3.

**Comment by rychlis** [comment]
We have just released Sketch Extension 3.0.0-prerelease4 with Sketch 3.5 support. Please download the latest version from https://github.com/avocode/avocode/blob/master/sketch-extension-beta.md#installation--update

We're working on updating the older Sketch Plugin 2 however there has been some further issues with Sketch 3.5 unfortunately. You should be able to use the beta version fine though

**Comment by rychlis** [comment]
Follow #609 for more updates about older Sketch plugin, I am closing this issue as a duplicate, thanks for the report

If you have issues with the prerelease extension, please report them as new issues
